packages/vue/README.md

```diff
@@ -0,0 +1,256 @@
+# Neutrino Vue Preset
+
+[![NPM version][npm-image]][npm-url]
+[![NPM downloads][npm-downloads]][npm-url]
+[![Join the Neutrino community on Spectrum][spectrum-image]][spectrum-url]
+
+## Features
+
+- Zero upfront configuration necessary to start developing and building a Vue web app
+- Modern Babel compilation.
+- Extends from [@neutrinojs/web](https://neutrino.js.org/packages/web)
```

Comment:
Copy this file to `docs/packages/vue/README.md` then replace these links with relative ones in that file, e.g.

`https://neutrino.js.org/packages/web` -> `../web`
`https://neutrino.js.org/project-layout` -> `../../project-layout`